rust/main/utils/run-locally/src/invariants/termination_invariants.rs

```diff
@@ -128,11 +136,11 @@ pub fn termination_invariants_met(
 
     // TestSendReceiver randomly breaks gas payments up into
     // two. So we expect at least as many gas payments as messages.
-    if gas_payment_events_count < total_messages_expected {
+    if gas_payment_events_count < total_messages_expected + sol_messages_with_non_matching_igp {
```

Comment:
I wonder if we can include `sol_messages_with_non_matching_igp` into `total_messages_expected`? Or is it not possible due to some other invariants?

Reply:
`total_messages_expected` is checked against at multiple places, like `delivered_messages_scraped`. I have added a comment "this is total messages expected to be delivered" to avoid any confusion.